packages/volto/src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -600,19 +614,33 @@ class Toolbar extends Component {
                 {!this.props.hideDefaultViewButtons && (
                   <button
                     className="user"
-                    aria-label={this.props.intl.formatMessage(
-                      messages.personalTools,
-                    )}
+                    aria-label={
+                      userHasRoles(this.props.user, [
+                        'Site Administrator',
+                        'Manager',
+                      ])
```

Comment:
I discussed this with Wagner yesterday and can give explanations on the decision to make it this way, since it was my suggestion to keep it like this.

The baseline is: it's the same check that is done for showing the "Site setup" button (see the PersonalTools component), so it made sense to me to be consistent.

The expanded answer would be that I would create a separate issue for this. In fact, the way to check this via restapi exists and resides in the presence of the site setup action in the actions that we get via the expanders from the content request. Therefore, the proper fix that I would suggest doing in a separate PR would be to actually rewrite the whole PersonalTools component to use the proper list of actions instead of the hardcoded one and fix this label accordingly in that PR.

Reply:
Great to hear it was already thought about, lets move this improvement to a separate issue then!